xpk.py

```diff
@@ -120,6 +121,161 @@
                 name: dshm-2
 """
 
+pw_workload_create_yaml = """apiVersion: jobset.x-k8s.io/v1alpha2
```

Comment:
This yaml would fit nicely in the pathways_xpk_helpers.py module since it is pathways specific.

Reply:
How would moving the config file create a circular dependencies?

Reply:
Discussed offline :)